### PR TITLE
fix: Markdownlint cleanup for ES

### PR DESCRIPTION
**Pull request by nschonni**
### Description

Cleanup some more code spans stuck to text and links after the converison

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
This is a long one, it seems good to me. it will be great to have a second review.

**Review by Graywolf9 — APPROVED** [review]
As allways thank you so much for your time and help! @nschonni 

And thank you so much for the quick review @JuanVqz

**Pull request merged**
